.vscode/tasks.json

```diff
@@ -0,0 +1,61 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Build (Jekyll)",
+      "type": "shell",
+      "command": "bundle",
+      "args": [
+        "exec",
+        "jekyll",
+        "build"
+      ]
+    },
+    {
+      "label": "Build (NodeJS)",
+      "type": "shell",
+      "command": "npm",
+      "args": [
+        "run",
+        "build"
+      ]
+    },
+    {
+      "label": "Build and watch (Jekyll)",
+      "type": "shell",
+      "command": "bundle",
+      "args": [
+        "exec",
+        "jekyll",
+        "serve",
+        "--force_polling",
+        "--livereload"
+      ]
+    },
+    {
+      "label": "Build and watch (NodeJS)",
+      "type": "shell",
+      "command": "npm",
+      "args": [
+        "run",
+        "watch:js"
+      ]
+    },
+    {
+      "label": "Build and watch (Jekyll + NodeJS)",
+      "dependsOn": [
+        "Build and watch (NodeJS)",
+        "Build and watch (Jekyll)"
+      ],
+      "group": {
+        "kind": "build",
+        "isDefault": true
+      },
+      "runOptions": {
+        "runOn": "folderOpen"
+      }
```

Comment:
I found that automatically starting tasks is not an ideal choice. If the project is opened using `Reopen in container`, task failures will occur.

<img width="1123" alt="Screenshot 2024-07-07 at 06 47 33" src="https://github.com/cotes2020/jekyll-theme-chirpy/assets/11371340/5cb9a26e-2648-45ba-8998-a7fdc214a76e">

Reply:
That's correct. We cannot run the build command before installing dependencies. I will check if there is a workaround, as I would prefer to run this default command automatically. I would even live with this behavior, but we will see.

Reply:
Please do not force everyone to manually set this to get rid of the auto-start error. Also, this configuration is unnecessary. In terms of workflow, it also duplicates the work of the initialization script. That is, every time you re-import or recompile the devcontainer, `npm run build` runs twice.

**First build**: 
https://github.com/cotes2020/jekyll-theme-chirpy/blob/e3a78b6243f7056105d72185bb6e94b436834e5b/.devcontainer/post-create.sh#L6

**Second build**:

The task `Start watch (Jekyll + NodeJS)`